dungeon/build.gradle

```diff
@@ -13,10 +13,10 @@ dependencies {
 
 
 sourceSets.main.java.srcDirs = ['src/', "$projectDir/build/generated-src/antlr/main/"]
-sourceSets.main.resources.srcDirs = ['assets/', {project(':game').sourceSets.main.resources.srcDirs}]
+sourceSets.main.resources.srcDirs = ['assets/', '../game/assets/']
 
 sourceSets.test.java.srcDirs = ['test/']
-sourceSets.test.resources.srcDirs = ['test_resources/']
+sourceSets.test.resources.srcDirs = ['test_resources/', '../game/test_resources', 'assets/', '../game/assets/']
```

Comment:
was? wieso hängen tests von den normalen assets ab?

Reply:
ich denke, weil in manchen tests teile des spiels verwendet bzw. instantiiert werden.

Reply:
ja, das ist wahr. aber ich denke, die überlegung ist gewesen, dass man sonst "doppelte" assets gehabt hätte